script.js

```diff
@@ -1,4 +1,228 @@
+var koreanGameMode = function (playerChoice, compChoice, playerChoiceFormatted, compChoiceFormatted){
+if (playerChoice == compChoice){
+  winningPlayer = currentWinner
+  var outputMessage = `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br> It's a Draw<br> Current Winner is: ${winningPlayer}<br>${winningPlayer} wins!`
+  return outputMessage
+  }
+if ((playerChoice == SCISSORS && compChoice == PAPER) || (playerChoice == PAPER && compChoice == STONE) || (playerChoice == STONE && compChoice == SCISSORS )){
+  currentWinner = userName
+  var outputMessage = `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${winMessage}<br>${currentWinner}: "muk-jji-ppa!`
+  return outputMessage
+  }
+if ((playerChoice == SCISSORS && compChoice == STONE)|| (playerChoice == PAPER && compChoice == SCISSORS) || (playerChoice == STONE && compChoice == PAPER) ){
+  currentWinner = 'Computer';
+  var outputMessage= `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${loseMessage}<br>${currentWinner}: "muk-jji-ppa!"`
+  return outputMessage
+  }
+};
+//Function for Computer Game Mode
+var computerGameMode = function (playerChoice, compChoice, playerChoiceFormatted, compChoiceFormatted){
+  totalGamesPlayed = totalGamesPlayed+1
+if (playerChoice == compChoice){
+  totalDraw = totalDraw+1
+  var gameStatus = scoreBoard(userName, playerScore, compScore, totalGamesPlayed,totalDraw)
+  var outputMessage = `It's a draw! 😃 <br> You chose ${playerChoiceFormatted} <br> The Computer chose ${compChoiceFormatted}<br><br> ${gameStatus}}`
+  return outputMessage
+  }
+
+if ((playerChoice == SCISSORS && compChoice == PAPER) || (playerChoice == PAPER && compChoice == STONE) || (playerChoice == STONE && compChoice == SCISSORS )){
+  playerScore = playerScore+1;
+  var gameStatus = scoreBoard(userName, playerScore, compScore, totalGamesPlayed,totalDraw)
+  var outputMessage = `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${winMessage}<br><br> ${gameStatus}`
+  return outputMessage
+  }
+if ((playerChoice == SCISSORS && compChoice == STONE)|| (playerChoice == PAPER && compChoice == SCISSORS) || (playerChoice == STONE && compChoice == PAPER) ){
+  compScore =compScore+1;
+  var gameStatus = scoreBoard(userName, playerScore, compScore, totalGamesPlayed,totalDraw)
+  var outputMessage= `The computer chose ${compChoiceFormatted}<br> You chose ${playerChoiceFormatted}<br><br>${loseMessage}<br><br> ${gameStatus}`
+  return outputMessage
+  }
+}
+//Function for Random number for the computer generated choice
+var compRandomChoice = function(){
+  var randomInt = Math.random() * 3;
+  var randomNum = Math.floor(randomInt);
+  if (randomNum == 0){
+    return 'scissors'
+  }
+  if (randomNum == 1){
+    return 'paper'
+  };
+  if (randomNum == 2){
+    return 'stone'
+  }
+};
+
+// function to format the choices to emoji
```

Comment:
Ah love this! This is an example of what we call a helper function where we abstract logic into its own block of code and run the function when needed anytime in our app to perform some kind of change excellent!